dcos/marathon.py

```diff
@@ -54,7 +54,11 @@ def _to_error(response):
     if isinstance(response, Error):
         return DefaultError(_default_marathon_error(response.error()))
 
-    message = response.json().get('message')
+    try:
+        message = response.json().get('message')
+    except ValueError as e:
```

Comment:
There aren't really docs about the message spec, but its mentioned here: https://mesosphere.github.io/marathon/docs/rest-api.html